File: app/src/main/java/io/github/gmazzo/android/livewallpaper/weather/engine/scenes/SceneCloudy.java
package io.github.gmazzo.android.livewallpaper.weather.engine.scenes;

import android.content.Context;

import javax.microedition.khronos.opengles.GL11;

import io.github.gmazzo.android.livewallpaper.weather.R;

public class SceneCloudy extends SceneClear {

    public SceneCloudy(Context ctx, GL11 gl) {
        super(ctx, gl, R.drawable.bg1);
    }

}

File: app/src/main/kotlin/io/github/gmazzo/android/livewallpaper/weather/engine/scenes/SceneCloudy.kt
package io.github.gmazzo.android.livewallpaper.weather.engine.scenes

import io.github.gmazzo.android.livewallpaper.weather.R
import javax.inject.Inject

class SceneCloudy @Inject constructor(
    dependencies: SceneDependencies,
) : Scene(
    dependencies,
    background = R.drawable.bg1,
    withSunAndMoon = true,
    withStars = true
)
